twilight-model/src/application/interaction/metadata.rs

```diff
@@ -28,9 +29,19 @@ pub struct InteractionMetadata {
     /// ID of the original response message, present only on follow-up messages.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub original_response_message_id: Option<Id<MessageMarker>>,
+    /// The ID of the message the command was run on, present only on
+    /// message command interactions.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub target_message_id: Option<Id<MessageMarker>>,
+    /// The user the command was run on, present only on user command
+    /// interactions.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub target_user: Option<User>,
```

Comment:
Just some little nit pick, to align with the comment style guidelines. 
```suggestion
    /// ID of the message the command was run on, present only on
    /// message command interactions.
    #[serde(skip_serializing_if = "Option::is_none")]
    pub target_message_id: Option<Id<MessageMarker>>,
    /// User the command was run on, present only on user command
    /// interactions.
    #[serde(skip_serializing_if = "Option::is_none")]
    pub target_user: Option<User>,
```